### ISSUE TITLE
python: can't open file 'server.py': [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by anikettiwariii** [insidesherpa/JPMC-tech-task-1-PY3#153]
I cloned the repository but still cannot open the file

**Comment by apurvadey** [comment]
@othersbet 
Try this: python server3.py or python3 server3.py
Worked for me

**Comment by anikettiwariii** [comment]
@apurvadey 
This worked but I got another error this time. No issues in server side but I get errors in client side- "No connection could be made because the target machine actively refused it." Do you know how to solve this? Thanks!

**Comment by apurvadey** [comment]
@othersbet 
Yes, just restart the server terminal and start the server again with the above command. And then, in the client terminal, give your command. This worked for me, hope it helps.

**Comment by anikettiwariii** [comment]
@apurvadey
Unfortunately, I am still getting the same error. I did everything as told in the pdf but it still won't work.

**Comment by apurvadey** [comment]
@othersbet 
Can you share any screenshot of both the terminals after starting afresh?

**Comment by anikettiwariii** [comment]
@apurvadey
![Server](https://user-images.githubusercontent.com/67806630/87230980-f046b100-c3d0-11ea-9361-f9bf44cbdc67.png)
![Client](https://user-images.githubusercontent.com/67806630/87230982-f2a90b00-c3d0-11ea-902e-00583962ee3b.png)

**Comment by apurvadey** [comment]
@othersbet 
Okay, got you!
You need to change the port 8080 to 8085 in server3.py and client3.py (if you are using server.py or client.py, the solution will be the same). change the port 8080 to 8085 where ever you encounter that in the whole file. and then try again.

**Comment by anikettiwariii** [comment]
@apurvadey
Did that but still same error :(

**Comment by apurvadey** [comment]
@othersbet 
Please ensure that you run the server before the client. This is the most apt reason why a ConnectionRefusedError is occuring.
You can refer [this thread](https://github.com/Lawouach/WebSocket-for-Python/issues/130)

**Comment by poojasn-github** [comment]
I cloned the repository but still cannot open the file client side is working but server side is not

**Comment by manuelkwizera** [comment]
Is this a proper output i'm supposed to get after typing `python server3.py`

![git](https://user-images.githubusercontent.com/66922751/88294009-5ad4e680-cd19-11ea-8317-eafbfac37b24.jpg)

**Comment by apurvadey** [comment]
@KwizeraEmmanuel Yes, it is valid. This means it is successfully communicating with client side, hence receiving queries.

**Comment by apurvadey** [comment]
@poojasn-github Can you please elaborate?

**Comment by subhanshu04** [comment]
> @othersbet
> Try this: python server3.py or python3 server3.py
> Worked for me


i tried this also but its not showing anything instead cmd is returning the same command 
![cdm](https://user-images.githubusercontent.com/64693472/99401923-c2d5bb00-290e-11eb-8070-2998435a9165.JPG)
